.github/workflows/e2e-test.yml

```
@@ -22,6 +22,7 @@ jobs:
   test:
     name: E2E test
     runs-on: ${{ inputs.runner }}
+    timeout-minutes: 60
```

```suggestion
    timeout-minutes: 90
```
Sorry, there are some e2e combos that take more than hour, namely in optional periodics.